Lesson11/lesson11_2.cpp

```diff
@@ -0,0 +1,44 @@
+// Lesson 11 2
+#include <iostream>
+
+const int SIZE = 30;
+enum class SortingDirection { ascending, descending };
+
+bool isSorted(const int* arr, int size, SortingDirection direction){
+    for(int i = 0; i < size-1; i++){
+        if(arr[i] > arr[i + 1] && direction == SortingDirection::ascending){
+            return false;
+        }
+        if(arr[i] <= arr[i + 1] && direction == SortingDirection::descending){
+            return false;
+        }
```

Comment:
you're checking sorting direction on every loop iteration. Please, check it only once

Reply:
fixed